File: Dasar-dasar Pemrograman/Semester 1/Python/Rekursi/uraifaktorial.py
param = input("Input nilai: ")
faktor = 1

desc = "faktorial("+param+") = 1*"
for i in range(1, int(param) + 1):
    if i == int(param):
        desc += str(i)+" = "
    
    else:
        desc += str(i)+"*"

    faktor *= i

print(desc+str(faktor))

File: Dasar-dasar Pemrograman/Semester 1/PHP/Rekursi/uraifaktorial.php
<?php 

echo "Masukkan nilai: ";
$param = trim(fgets(STDIN));
$faktor = 1;

$desc = "faktorial(".$param.") = 1*";
for ($i = 1; $i < $param + 1; $i++) {
    if ($i == $param){
        $desc .= $i." = ";
    } else {
        $desc .= $i."*";
    }

    $faktor *= $i;
}

echo $desc;
echo $faktor;
